Sources/Quick/World.swift

```diff
@@ -38,6 +38,11 @@ final internal class World: NSObject {
     */
 
     internal var currentExampleMetadata: ExampleMetadata?
+    
+    /**
+        The descriptor of the current context.
+    */
```

Comment:
Nitpick: I rather have this comment removed or elaborated more. It currently doesn't provide extra value beyond the variable name.